### ISSUE TITLE
Loss trends of Pretrain LayoutLMv3

### ISSUE THREAD

**Issue by kash203** [microsoft/unilm#952]
## Describe
Model I am using (UniLM, MiniLM, LayoutLM ...): LaoyutLMv3

I pre-trained LayoutLMv3 base model, it seems convergence begins at high MLM/MIM loss value.
MLM/MIM loss convergence begins at about 6~7, WPA loss convergence at 0.5. So Total loss is about 12~15.
These loss values seem high relative to normal MLM losses, I think it is strange.

**Can you provide log of pre-training on LayoutLMv3 base model?**

There is one more thing that I find strange, when learning MLM with span masking, 
it seems that top1 cannot be restored for sentences after the input length of the data after masking. 
(It was the same even when it was overtrained. and length of `attention_mask` is aligned with the data before masking.)

Specifically, there was a tendency for the last character of the correct sentence to be top1 for the reconstructed words after the input length after masking.
(Although slightly close values may be entered after top2)

for example: 
```
original data        : A B C D E F
masked data          : A [MASK] F
model output (top 1) : A B C F F F
```

I would like to check whether the above behavior is correct.

## Conditions:
- Input length to the transformer is about 709 that contains word(512) and image(197)
  - batch size per GPU: about 50
  - using gradient accumulation: 10=> 50 x 4 x 10 =2,000

**Comment by yash0307** [comment]
Hi, where is the code for pre-training?

**Comment by kash203** [comment]
Hi @yash0307,
I think it is not released, so I'm implementing it myself by referring to papers.
So I want to check the loss trends, just like checking answers.

**Comment by vanpersie32** [comment]
> hello, could you please make your code public

**Comment by hieutt196** [comment]
Hi, could you please give me some information such as: number of images, the training time, ...?
